app/uk/gov/hmrc/gform/eval/EvaluationResults.scala

```diff
@@ -508,6 +547,23 @@ case class EvaluationResults(
           case _              => unsupportedOperation("String")(expr)
         }
       case RemoveSpaces(fcId) => removeSpaces(fcId, recData)
+      case NumberedList(fcId)
+          if evaluationContext.addressLookup(fcId.baseComponentId) || evaluationContext
+            .overseasAddressLookup(fcId.baseComponentId) =>
+        val list = atlAddresses(fcId)
+        StringResult(govukList(list, false))
+      case NumberedList(fcId) =>
+        val list = atlValues(fcId)
+        StringResult(govukList(list, false))
+      case BulletedList(fcId)
+          if evaluationContext.addressLookup(fcId.baseComponentId) || evaluationContext
+            .overseasAddressLookup(fcId.baseComponentId) =>
+        val list = atlAddresses(fcId)
+        StringResult(govukList(list, true))
+
+      case BulletedList(fcId) =>
+        val list = atlValues(fcId)
```

Comment:
Replace `atlValues(fcId)` with `loop(FormCtx(fcId))` and process resulting `ListResult` accordingly.

Note: `def atlValues` should be eliminated